__tests__/commands/openapi/upload.test.ts

```diff
@@ -50,6 +52,26 @@ describe('rdme openapi upload', () => {
       mock.done();
     });
 
+    it('should create a new YAML API definition in ReadMe', async () => {
+      const mock = getAPIv2Mock({ authorization: `Bearer ${key}` })
+        .get(`/versions/${version}/apis`)
+        .reply(200, { data: [] })
+        .post(`/versions/${version}/apis`, body =>
+          body.match(`form-data; name="schema"; filename="${slugifiedYamlFile}"\r\nContent-Type: application/x-yaml`),
+        )
+        .reply(200, {
+          data: {
+            upload: { status: 'done' },
+            uri: `/versions/${version}/apis/${slugifiedYamlFile}`,
+          },
+        });
+
+      const result = await run(['--version', version, yamlFile, '--key', key]);
+      expect(result.stdout).toContain('was successfully created in ReadMe!');
```

Comment:
mind making this a snapshot assertion?

```suggestion
      expect(result).toMatchSnapshot();
```